src/wallet/wallet.cpp

```diff
@@ -1625,7 +1625,11 @@ isminetype CWallet::IsMine(const CScript& script) const
     }
 
     // Legacy wallet
-    if (IsLegacy()) return GetLegacyScriptPubKeyMan()->IsMine(script);
+    if (IsLegacy()) { // Blank legacy wallets have a null spkm
```

Comment:
In 30f701317b4a674fea422766ac3ee179c9c6f554 "wallet: fix, detect blank legacy wallets in IsLegacy"

`IsLegacy()` is unnecessary here now since `GetLegacyScriptPubKeyMan()` also checks for `WALLET_FLAG_DESCRIPTORS`.

Reply:
done